src/components/actions/scheduler.ts

```diff
@@ -2,10 +2,18 @@ import DBConnect from '../databaseConnect'
 import scheduler from 'node-schedule'
 import config from '../../config'
 import { DateTime } from 'luxon'
+import fs from 'fs'
+import { filesFolder } from '../files/fileHandler'
+import path from 'path'
+import { getAppEntryPointDir } from '../utilityFunctions'
+import { crawlFileSystem } from '../utilityFunctions'
+import { deleteFile } from '../files/deleteFiles'
 import createBackup from '../exportAndImport/backup'
 
 // Dev config option
 const schedulerTestMode = false // Runs scheduler every 30 seconds
+const basePath: string = path.join(getAppEntryPointDir(), filesFolder) // declares the file storage path
```

Comment:
This "basePath" is already defined in `./constants.ts` -- you can just import "FILES_FOLDER" instead and it includes the full path.